theme.css

```diff
@@ -4090,7 +4093,11 @@ body.anp-card-layout .workspace-split .workspace-leaf-content:not([data-type=fil
   margin-bottom: var(--anp-card-layout-padding, 10px);
 }
 body.anp-card-layout.anp-card-shadows .workspace-split .workspace-leaf-content, body.anp-card-layout.anp-card-shadows.anp-card-layout-actions .side-dock-actions, body.anp-card-layout.anp-card-shadows.anp-card-layout-filebrowser .workspace-split .workspace-leaf-content[data-type=file-explorer] {
-  box-shadow: 0 3px 4px 0px rgba(0, 0, 0, 0.05);
+  box-shadow: 0px 3px 4px 0px rgba(0, 0, 0, 0.05);
+}
+body.anp-card-layout.anp-card-shadows .menu {
+  box-shadow: 3px 3px 4px 0px rgba(0, 0, 0, 0.05), -3px 3px 4px 0px rgba(0, 0, 0, 0.05);  /*add shadow for menu*/
+
```

Comment:
I think removing this would be a better choice since shadows only really apply to the actual workspace elements and the menu is separate from the actual workspace itself